cron/cron.go

```diff
@@ -105,23 +107,24 @@ func runJob(context *crontab.Context, command string, jobLogger *logrus.Entry) e
 	return nil
 }
 
-func StartJob(wg *sync.WaitGroup, context *crontab.Context, job *crontab.Job, exitChan chan interface{}, cronLogger *logrus.Entry) {
+// StartJob starts the cron job.
+func StartJob(wg *sync.WaitGroup, context *crontab.Context, job *crontab.Job, exitChan chan interface{}, cronLogger *logrus.Entry, overlapping bool) {
 	wg.Add(1)
 
 	go func() {
 		defer wg.Done()
 
-		var cronIteration uint64 = 0
+		var cronIteration uint64
 		nextRun := time.Now()
 
-		// NOTE: this (intentionally) does not run multiple instances of the
-		// job concurrently
+		// NOTE: if overlapping is disabled (default), this does not run multiple
+		// instances of the job concurrently
 		for {
 			nextRun = job.Expression.Next(nextRun)
 			cronLogger.Debugf("job will run next at %v", nextRun)
 
 			delay := nextRun.Sub(time.Now())
-			if delay < 0 {
+			if delay < 0 && !overlapping {
```

Comment:
Unless the job takes longer to _start_ than the frequency (very unlikely), delay should never be `< 0` if overlapping jobs are allowed, so is there a reason to include this here?